_posts/2018-08-24-bitcoin-abc-vision.md

```diff
@@ -0,0 +1,26 @@
+---
+layout: post
+title: The Bitcoin ABC Vision
+subtitle: Proposed Technical Roadmap for Bitcoin Cash
+multiLangId: 2018-08-24-bitcoin-abc-vision
+lang: en
+---
+
+The goal of Bitcoin ABC is to help create sound money that is usable by everyone in the world. We believe this is a civilization-changing technology which will dramatically increase human freedom and prosperity.
+
+Our role in furthering this goal is to produce high-quality software that serves the needs of the community of users, miners and merchants. We strive for continuous technical improvement, to produce a reliable product that will provide a solid foundation for Bitcoin Cash.
+
+The recent release of Bitcoin ABC 0.18.0 was the latest step along this path. It continues the process of careful and incremental technical progress.
+
+In order to plan for the future, we base our plans on sound technical engineering to make design decisions. Based on this approach, we believe the future development of Bitcoin Cash should focus on three areas:
+
+Improving 0-conf transactions. We want to ensure that the payment experience instant and reliable, without requiring any changes to the consensus rules.
+ 1. Implementing technical measures to enable massive on-chain scaling. The fundamental design of Bitcoin Cash is sound, but there are some technical improvements that must be made to enable Bitcoin Cash to scale to a world-wide currency. We are working to remove immediate bottlenecks such as the mempool admittance code and mining RPC, as well as laying the groundwork to enable massive on-chain scaling well into the future.
+ 2. Improving fungibility. As a cash system, Bitcoin Cash should embrace methods to improve its fungibility.
+ 3. We believe that the basic design of Bitcoin Cash is sound, and does not need any radical change. However, this does not mean it is perfect. It is prudent to make incremental improvements to the system with technically sound design and careful engineering. By implementing optimizations and protocol upgrades, we can enable peer-to-peer digital cash to scale many orders of magnitude beyond current limits.
+
+[Bitcoin ABC Roadmap](https://github.com/Bitcoin-ABC/bitcoinabc.org/blob/master/img/abc-roadmap-whitebg.png)
```

Comment:
I think this needs to be prepended by a `!`. and please include the image file as part of this PR